R/class-workbook-wrappers.R

```diff
@@ -3008,6 +3002,11 @@ wb_add_comment <- function(
   ) {
 
   assert_workbook(wb)
+
+  if (is.character(comment)) {
```

Comment:
This should accept more cases and maybe only exclude class comment? Why e.g. shouldn't it be possible to pass a number or a fmt_txt() down or dunno even a vector? People get strange ideas ... eh, with people I mean myself :)

Reply:
`is.character(fmt_txt("x")` is TRUE. So, this will be accepted. A number.. Not sure ! Maybe wait for user requests?

Reply:
I tested that fmt_txt inputs work. And they show correctly in Excel. I added an example in `wb_add_comment()`